backend/src/core/logger.ts

```diff
@@ -1,4 +1,13 @@
+import path from 'path'
 import winston from 'winston'
+import requestTracer from 'cls-rtracer'
+
+const getModuleLabel = (callingModule: NodeModule): string => {
+  // Remove the file extension from the filename and split with path separator.
+  const parts = callingModule.filename.replace(/\.[^/.]+$/, '').split(path.sep)
+  // Join the parts of the file path after build directory together
+  return path.join(...parts.slice(parts.lastIndexOf('build') + 1))
```

Comment:
if we trust that 'build' is indeed the name of the build directory, then this is equivalent
```js
const moduleName =  callingModule.filename
moduleName.substring(moduleName.lastIndexOf('build')+6, moduleName.lastIndexOf('.'))
```